js/dydx.js

```diff
@@ -0,0 +1,1252 @@
+                        'orders/{id}': 1, // variable, 250 requests per 10 seconds PER SYMBOL
+                        'active-orders': 1, // variable
+                    },
+                    'put': {
+                        'users': 60,
+                        'emails/send-verification-email': 3000,
+                    },
+                },
+            },
+            'commonCurrencies': {
+                //
+            },
+            'precisionMode': TICK_SIZE,
+            'fees': {
+                'trading': {
+                    'tierBased': false, // complicated tier system per coin
+                    'percentage': true,
+                    'maker': 0.05 / 100,
+                    'taker': 0.1 / 100,
+                },
+            },
+            'exceptions': {
+                'exact': {
+                    '404': ExchangeNotAvailable,
+                    '403': AuthenticationError,
+                    '400': ExchangeError,
+                    'dydx-api-key': AuthenticationError,
+                    'dydx-passphrase': AuthenticationError,
+                    'dydx-timestamp': BadRequest,
+                },
+                'broad': {
+                    'must be a valid string that is less than length:': BadRequest,
+                    'Cannot read property ': BadRequest,
+                    'signature must be a string': AuthenticationError,
+                    'must be a boolean': BadRequest,
+                    'Invalid number': BadRequest,
+                    'expiration must be an ISO8601 string': BadRequest,
+                    'Invalid signature for order': InvalidOrder,
+                    'No order for market: ': InvalidOrder,
+                    'status must be a valid array of subset': BadRequest,
+                    'API key not found': AuthenticationError,
+                    'api-key must be a valid UUID in headers': AuthenticationError,
+                    'Unauthorized': AuthenticationError,
+                    'passphrase must be a valid base64url string in headers that is 20 characters long': AuthenticationError,
+                    'timestamp must be a valid ISO string in headers': BadRequest,
+                    'ethereum address must be a valid string in headers': AuthenticationError,
+                    // old
+                    'See /corsdemo for more info': AuthenticationError,
+                    'Invalid signature for onboarding request': AuthenticationError,
+                    'Invalid signature for ApiKey request': AuthenticationError,
+                    'Not Found': ExchangeNotAvailable,
+                    'Invalid value': ExchangeError,
+                    'AccountNotFoundError': AuthenticationError,
+                    'market must be a valid market': BadSymbol,
+                    'Invalid market': BadSymbol,
+                    'Order is below minimum size of 20000000000000000000': InvalidOrder,
```

Comment:
I would remove the amount from this string leaving just `Order is below minimum size of` and would put that to the broad-exceptions subdict. Cause they can change their limits anytime.